website/static/js/pages/component-settings-page.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var $ = require('jquery');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+var $osf = require('js/osfHelpers.js');
+var getConfirmationCode = require('js/projectSettings.js').getConfirmationCode;
```

Comment:
This should be extracted from projectSettings and imported separately by the projectSettings file and component-settings-page. Console is throwing an error because the project overview page is now loading the whole projectSettings page and running JS that it doesn't have the correct data to execute.